moped-database/views/project_list_view.sql

```diff
@@ -81,7 +81,7 @@ AS WITH project_person_list_lookup AS (
       FROM moped_proj_phases phases
       WHERE true
         AND phases.project_id = mp.project_id 
-        AND phases.phase_name = 'construction'::text
```

Comment:
i wonder if we might want to leave this as-is (matching on the `phase_name` text). it feels fragile to rely on the ID sequence.

Reply:
edit: ignore me! phase_name is not available, and i don't think we need to take the extra step of filtering on the `moped_phases.phase_name`